waspc/src/Wasp/Generator/Common.hs

```diff
@@ -38,3 +42,16 @@ npmVersionRange =
 
 prismaVersion :: SV.Version
 prismaVersion = SV.Version 3 15 2
+
+-- | Adds .cmd extension to callable npm command if ran on Windows, since npm does note have .exe on Windows.
+--   The reason explained here: https://stackoverflow.com/questions/43139364/createprocess-weird-behavior-with-files-without-extension
+oSSpecificNpm :: String
+oSSpecificNpm = "npm" ++ if os /= "mingw32" then "" else ".cmd"
+
+-- | Changes an npm command to a cmd.exe command on Windows only. Calling npm from API causes troubles.
```

Comment:
Similar like above, this is addressing implementation details. I would make sure this comment talks about what function does from the outside, now how it does it, and details can go inside of function. We might not even need this comment if it is clean what it does from the function name + signature.

Reply:
moved comments into implementation